src/frameworks/gatsby.json

```diff
@@ -23,6 +23,11 @@
     "AWS_LAMBDA_JS_RUNTIME": "nodejs14.x",
     "NODE_VERSION": "14"
   },
+  "logos": {
+    "default": "Gatsby_Default.svg",
+    "light": "Gatsby_Black.svg",
+    "dark": "Gatsby_White.svg"
```

Comment:
I added `.png`s to the folder if that's what we prefer to use, but opted to use SVGs for the initial pass.

Question around organization: Do we want to have each framework have a folder for their logos within `logos`? i.e: `frameworks > logos > gatsby > default.svg`?

Reply:
Following up from our Slack conversation - initially the thought was to aim to provide a consistent way of getting access to the logo across our UI project, the CLI, and the build tool (which all have this project as a dependency) so that, but given the main goal of adding these icons, transitioning towards using links to fetch the icons makes more sense.

Slack thread for context: https://netlify.slack.com/archives/C01TKAEBP3Q/p1658838039556039